src/site/pages/digital/src/containers/App/index.tsx

```diff
@@ -88,7 +88,7 @@ const exampleCircuits = exampleConfig.examples.map((example) =>
 );
 
 export const App = ((store: AppStore) => {
-    const canvas = createRef<HTMLCanvasElement>();
+    const canvas = createRef<HTMLCanvasElement>() as MutableRefObject<HTMLCanvasElement>;
```

Comment:
Why?

Reply:
In the line `<MainDesigner info={info} canvas={canvas} />` the error 
```
Type 'RefObject<HTMLCanvasElement>' is not assignable to type 'MutableRefObject<HTMLCanvasElement>'.
  Types of property 'current' are incompatible.
    Type 'HTMLCanvasElement | null' is not assignable to type 'HTMLCanvasElement'.
      Type 'null' is not assignable to type 'HTMLCanvasElement'.ts(2322)
index.tsx(27, 5): The expected type comes from property 'canvas' which is declared here on type 'IntrinsicAttributes & Props'
```
is produced

Reply:
Then probably change the type of `Props` in in `MainDesigner` to take in a `RefObject` instead?